app/queries/get_content_item_ids_with_fallbacks.rb

```diff
@@ -9,25 +9,35 @@ def self.call(content_ids, locale_fallback_order: ContentItem::DEFAULT_LOCALE, s
       content_items = ContentItem.arel_table
       states = State.arel_table
       translations = Translation.arel_table
+      unpublishings = Unpublishing.arel_table
 
-      fallbacks = cte(
-        content_items.project(
-          content_items[:id],
+      fallback_scope = content_items.project(
+        content_items[:id],
             content_items[:content_id],
           )
           .join(states).on(states[:content_item_id].eq(content_items[:id]))
           .join(translations).on(translations[:content_item_id].eq(content_items[:id]))
           .where(content_items[:content_id].in(content_ids))
           .where(content_items[:document_type].not_in(::ContentItem::NON_RENDERABLE_FORMATS))
-          .where(states[:name].in(state_fallback_order))
           .where(translations[:locale].in(locale_fallback_order))
-          .order(
-            order_by_clause(:states, :name, state_fallback_order),
-            order_by_clause(:translations, :locale, locale_fallback_order)
-          ),
-        as: "fallbacks"
+
+      if state_fallback_order.include?("withdrawn")
```

Comment:
We've used `withdrawn` here as a state because although it currently isn't a 'state' it should be. `unpublished` has unfortunately been overloaded (as it also is in Whitehall) and is actually a set of three or four states. We're explicitly targeting `withdrawn` content here rather than `redirected`, `gone` etc. At some point we need to revisit how unpublishing states are implemented but for now this at least should clarify the intent.